src/app/auth/signin/page.tsx

```diff
@@ -0,0 +1,86 @@
+'use client'
+
+import { MainColors } from '@/shared/constants'
+import { Button, Card, Center, Divider, Group, Text, Title } from '@mantine/core'
+import { BuiltInProviderType } from 'next-auth/providers/index'
+import { ClientSafeProvider, LiteralUnion, getProviders, signIn, useSession } from 'next-auth/react'
+import { useEffect, useState } from 'react'
+
+export default function SignIn() {
+  const { data } = useSession()
+  const [providers, setProviders] = useState<Record<
+    LiteralUnion<BuiltInProviderType, string>,
+    ClientSafeProvider
+  > | null>(null)
+  useEffect(() => {
+    async function init() {
+      setProviders(await getProviders())
+    }
+    init()
+  }, [])
+
+  console.log(providers)
+
+  return (
+    <Center>
+      <Card w="300px" shadow="sm" radius="xl" p={20} bg={MainColors.PinkWhiteBG} h="30vh">
+        <Title order={3} mt={10} ta="center">
+          Welcome to Jax
+        </Title>
+        <Text c="darkgrey" ta="center" mt={20}>
```

Comment:
Minor nitpick, I think the color should change. Either a darker grey or just black. If we go with black, maybe have the text a little smaller?